docs/workflow/blocks/actions.rst

```diff
@@ -26,8 +28,14 @@ Supported properties
 - **buttons** (array) - the list of buttons. Each button in the list has the following properties:
 
   - **label** (string) - the text for the button label
+  - **id** (string) - the unique identifier for the button
   - **color** (string) - passed as the “color” attribute to the material button. Use one of the supported
     Material color values, such as “primary”, “warn”, “accent”. Leave as “default” to use the default button styling for a plain button.
+  - **enabledGetter** (string) - A value from the state or context. Based on this value, it generates an "enabled" property on the button.
+  The button will be shown as disabled (grey out) or enabled accordingly. The default value of this property is `true`. After 
+  the block is loaded, it will automatically add a property to the block json: `"enabled": true`.
+  If `enabledGetter` is set to `false` the automatic property will be then shown as following: `"enabled": false`.
+  - **valueGetters** (string) - A value that will be pass as context to whichever action is being executed.
```

Comment:
I think this needs to point out that it's a JMESPath mapping, that is evaluated like the Mapping block is, to determine if the button should be enabled or not. @gsambrotta

Reply:
@lukestanley Please suggest a text to add. I don't know much about the `valueGetters`. The text I wrote describe my knowledge of it.